assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -466,6 +466,8 @@ const baseSelectors = {
 
 		const isViewOnly = ! select( CORE_USER ).isAuthenticated();
 
+		// TODO The below logic is specific to unavailable custom dimensions for view only users
+		// who had user picked metrics. So move this to somewhere more appropriate.
```

Comment:
Minor copy-editing here 😅 

```suggestion
		// TODO: The below logic is specific to unavailable custom dimensions for view only users
		// who had user-picked metrics. So move this to somewhere more appropriate.
```

But more importantly: where should this be moved to? I'm not sure exactly what this comment means/what it's for. Should we create a follow-up issue for something? 🤔

Reply:
#9548 will cover this 💯  This will be moved to a callback function.